core/jsonld/auth.jsonld

```diff
@@ -22,6 +22,92 @@
           "@type": "xsd:string"
         }
       }
+    },
+    "Iden3StateInfo2023": {
+      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/auth.json-ld#Iden3StateInfo2023",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "@vocab": "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/state-info.md#",
```

Comment:
copy vocab from credentials to core folder also and change the url

Reply:
Done